formats/log/identifier.go

```diff
@@ -0,0 +1,14 @@
+package log
+
+import (
+	"crypto/sha256"
+	"fmt"
+)
+
+// ID returns the identifier to use for a log given the Origin
+// and the public key. This is the ID used to find checkpoints
+// for this log at distributors, and that will be used to feed
+// checkpoints to witnesses.
+func ID(origin string, key []byte) string {
```

Comment:
It feels a bit unexpected given the comment that this doesn't use the origin string passed in?

Reply:
We can optimize to remove the origin if we don't need it. I've added this signature to take both of these things for now so that we can continue to discuss what the implementation should be. It's clear to me that it certainly shouldn't be taking anything *more* than these though.